server/NewBankClientHandler.java

```diff
@@ -105,4 +129,51 @@ public void sendOutput(String s) {
 			e.printStackTrace();
 		}
 	}
+
+	private void showInitialMenu(){
+		out.println("+------------------------------------+");
+		out.println("|NewBank - Initial Menu              |");
+		out.println("+------------------------------------+");
+		out.println("|(1) - Login as existing customer    |");
+		out.println("|(2) - Register for a new account    |");
+		out.println("+------------------------------------+");
+	}
+
+	private void showCustomerMenu(Customer customer){
+		out.println("\nLoading...\n");
+		out.println("Log In Successful.\n");
+		out.println("Welcome to your NewBank account " + customer.getUsername()+".");
+		out.println("\n+---------------------------------------------------+");
+		out.println("|NewBank - Customer Menu                            |");
+		out.println("+---------------------------------------------------+");
+		out.println("|(1) - Show existing accounts and current balance   |");
+		out.println("|(2) - Create a new account with zero opening funds |");
+		out.println("|(3) - Move funds between your existing accounts    |");
+		out.println("|(4) - Pay an existing NewBank customer             |");
+		out.println("+---------------------------------------------------+");
+	}
+
+	//counts the frequency of usernames for which logins have been attempted and checks against maximum attempts set
+	private String userLogInFrequency(ArrayList<String> failedLogInUsers, int maxAttempts){
```

Comment:
This function doesn't really need `failedLogInUsers` as a parameter, because it's already part of the class, it can just access it